installers/nanopi-r4s/src/main.go

```diff
@@ -17,8 +17,9 @@ import (
 )
 
 const (
-	off int64 = 512 * 64
-	dtb       = "rockchip/rk3399-nanopi-r4s.dtb"
+	off   int64 = 512 * 64
+	board       = "nanopi-r4s"
```

Comment:
@frezbo Should I add this for the other boards too?

Also, the `MAINTAINERS.md`  is in place now.

Reply:
that would be nice

Reply:
Done.